packages/client/src/payments/types/paymentInstrumentData.types.ts

```diff
@@ -1,3 +1,13 @@
+export enum Channel {
```

Comment:
If these types are exclusive to pay by link, I would rename this to `PayByLinkChannel` and `PayByLinkContext` as `Channel` and `Context` are too generic.